integration-tests/src/test/java/org/zowe/apiml/integration/ha/SouthboundServicesRoutingTest.java

```diff
@@ -40,6 +41,7 @@
 @HATest
 public class SouthboundServicesRoutingTest {
     private GatewayServiceConfiguration gatewayServiceConfiguration;
+    private DiscoveryServiceConfiguration discoveryServiceConfiguration;
```

Comment:
It looks like we replicate the configuration and setup a lot across the HA classes could we maybe try to find a way to extract it?

Reply:
WIll do later.